tests/test_writing_methods.py

```diff
@@ -44,9 +94,12 @@ def test_051_add_account_and_sync_wrong_and_correct_pin(figo_session):
         task_state = figo_session.add_account_and_sync("de", wrong_credentials, BANK_CODE)
     except FigoPinException as pin_exception:
         task_state = figo_session.add_account_and_sync_with_new_pin(pin_exception, CREDENTIALS[1])
-    time.sleep(5)
-    assert isinstance(task_state, TaskState)
-    assert len(figo_session.accounts) == 3
+        assert isinstance(task_state, TaskState)
+        assert len(figo_session.accounts) == 3
+    except FigoException as figo_exception:
```

Comment:
I wrote that in the comment. If you use that test with the demo account, it returns an error without a code and that raises a `FigoException`.